app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
```

Comment:
Is it worth including in the commit message a note of how we generated those views?

```rb
rails generate devise:views
```

We want to make our commit narrative as helpful and clear as possible after all. We don't know if the reader is: 

- a complete newcomer to Rails (in which case this may be a useful clue that gems often provide their own rails generators, who knew?)
- a seasoned Rails dev, in which case no harm done